tests/unittest/_torch/attention/test_attention_backends.py

```
@@ -0,0 +1,242 @@
+                    if cfg.is_cross:
+                        yield (
+                            f"{cfg.id}-same_kv_{tag}",
+                            BackendCase(
+                                seq_lens_kv=_PHASES["ctx"]["seq_lens"], **_PHASES["ctx"], **base
+                            ),
+                        )
+                        yield (
+                            f"{cfg.id}-diff_kv_{tag}",
+                            BackendCase(
+                                seq_lens_kv=[
+                                    x * y for x, y in zip(_PHASES["ctx"]["seq_lens"], [2, 3, 6])
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)